charts/seed-bootstrap/charts/monitoring/templates/config.yaml

```
@@ -101,3 +101,21 @@ data:
     - job_name: prometheus
       static_configs:
       - targets: [ localhost:9090 ]
+
+    - job_name: vpa-recommender
+      kubernetes_sd_configs:
+        - role: pod
+          relabel_configs:
+            - source_labels: [__meta_kubernetes_pod_label_app]
+              regex: vpa-recommender
+              action: keep
+            - source_labels: [ __meta_kubernetes_service_port_name ]
```

Using `__meta_kubernetes_service_port_name` will probably not work since the via-recommender does not have a service.
```suggestion
            - source_labels: [__meta_kubernetes_pod_container_port_name]
```

See: https://prometheus.io/docs/prometheus/latest/configuration/configuration/#pod